packages/commercetools/theme/pages/Checkout/ThankYou.vue

```diff
@@ -0,0 +1,240 @@
+        <div class="banner__order-number">
+          <span>Order No.</span>
+          <strong>{{ order.number }}</strong>
+        </div>
+      </template>
+    </SfCallToAction>
+    <section class="section">
+      <div class="order">
+        <SfHeading
+          title="Your Purchase"
+          class="order__heading heading sf-heading--left"
+          :level="3"
+        />
+        <p class="order__paragraph paragraph">
+          You have successfully placed the order. You can check status of your
+          order by using our delivery status feature. You will receive an order
+          confirmation e-mail with details of your order and a link to track its
+          progress.
+        </p>
+        <div class="order__contact">
+          <SfHeading
+            :level="6"
+            class="heading sf-heading--left sf-heading--no-underline"
+            title="Primary contacts for any questions"
+          ></SfHeading>
+          <div class="contact">
+            <p class="contact__name">{{ address.name }}</p>
+            <p class="contact__street">{{ address.street }}</p>
+            <p class="contact__city">{{ address.city }}</p>
+            <p class="contact__email">{{ address.email }}</p>
+          </div>
+        </div>
+        <SfButton class="order__notifications-button button-size"
+          >Allow order notifications</SfButton
+        >
+      </div>
+      <div class="additional-info">
+        <div>
+          <SfHeading
+            title="Your Account"
+            class="heading sf-heading--left"
+            :level="3"
+          />
+          <p class="paragraph">
+            You can log to your account using e-mail and password defined
+            earlier. On your account you can edit your profile data, check
+            history of transactions, edit subscription to newsletter.
+          </p>
+        </div>
+        <div>
+          <SfHeading
+            title="What can we improve"
+            class="heading sf-heading--left"
+            :level="3"
+          />
+          <p class="paragraph">
+            Your feedback is important to us. Let us know what we could improve.
+          </p>
+          <SfButton
+            class="feedback-button color-secondary sf-button--full-width button-size"
+            >Send my feedback</SfButton
+          >
+        </div>
+      </div>
+    </section>
+    <nuxt-link to="/" class="sf-button back-button color-primary button-size">Go back to shop</nuxt-link>
+  </div>
+</template>
+
+<script>
+import { SfHeading, SfButton, SfCallToAction } from '@storefront-ui/vue';
+
+export default {
+  components: {
+    SfHeading,
+    SfButton,
+    SfCallToAction
+  },
+  setup(props, context) {
+    return {
+      address: {
+        name: 'Divante Headquarter',
+        street: 'St. Dmowskiego 17, 53-534',
+        city: 'Wroclaw, Poland',
+        email: 'demo@vuestorefront.io'
+      },
+      order: {
+        number: `#${context.root.$route.query.order}`
```

Comment:
What will happen when the order doesn't exist? The order number will be `#undefined`?